src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -907,7 +907,7 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
           justify-content: space-between;
         }
         .title h1 {
-          font-family: Roboto;
+          font-family: var(--ha-font-family-body);
```

Comment:
Please don't use only Roboto but the globals which handle all languages correctly.